test/testcommon/src/main/java/io/pravega/test/common/TestUtils.java

```diff
@@ -37,11 +37,19 @@
     private static final int BASE_PORT = 32768;
     private static final int MAX_PORT_COUNT = 28233;
     private static final Random RAND = new Random();
+    private static final String TLS_PROTOCOL_VERSION = "TLSv1.2,TLSv1.3";
```

Comment:
What is the difference between this and `SecurityConfigDefaults`?

Reply:
Initially used this because some of tests already imported `TestUtils`. Referred to `SecurityConfigDefaults` in all tests and removed this.